README.md

```diff
@@ -66,12 +66,13 @@ $ mergerfs.fsck -v -f manual /path/to/dir
 
 ### mergerfs.dup
 
-Duplicates files & directories across drives in pool.
-
-Copies files to drives with the most free space.
+Duplicates files & directories across drives in pool copying files to drives with the most free space.
 
 Run as `root`. Requires `rsync` to be installed.
 
+**Important Warning:** This tool is meant to only be used in archival data that never changes. Any change to a file that's been duplicate will not be replicated in it's copy even though mergerfs will update it's modified timestamp. This means you'll have two versions of the file and the one visable in the mergerfs pool can change on remount, aditionaly using mergerfs.dedup may cause loss of data since it will the first file it sees instead of the oldest has it only checks for timestamp and not content.
```

Comment:
A more accurate description is simply that the tool does not look for differences in the files. It merely ensures >=N files of that relpath exist and copies if not. We can add this note but I'd rather just change the code to add updating.

Reply:
My intent was to simply make it clear how the tool works now, since misunderstandings could potentially lead to data loss.

Reply:
I understand your intent but what you wrote is wrong regarding data loss and how the software works.